src/components/Content/Content.js

```diff
@@ -1,15 +1,214 @@
-import React from "react";
+import React, { useState } from "react";
+import DataService from "../../services/DataService";
+import { FormGroup, Label, Input, InputGroup, InputGroupAddon, InputGroupText } from "reactstrap";
 
 import "./Content.css";
 
+const formatSections = (data, activeSection, activeSubsection, setActiveSubsection, variables, setVariables) => {
```

Comment:
If you were to rewrite this for functional components you would write it as follows:
```suggestion
const formatSections = ({data, activeSection, activeSubsection, setActiveSubsection, variables, setVariables}) => {
```

```
// example of use in render
<formatSections data={...} activeSection={...} ... />
// or as function
{formatSections({data, activeSection, ...})}
```

Reply:
If you have a component whose props change frequently or pass many of their props downward you can also do the following:
```
const MyComponent = (props) => {
  return (
  <>
    <SomeComponent title={props.title} />
    <NeedyComponent {...props} title="childTitle"} />
  </>
  );
  }